### PR TITLE
scrum masters

### PR DESCRIPTION
**Pull request by durandom**

### CODE REVIEW

**Comment by durandom** [comment]
@tumido can we use peribolos-as-a-service for this repo?

File: github-config.yaml [hunk 1]
```diff
@@ -115,6 +115,16 @@ orgs:
         privacy: closed
         repos:
           community: admin
+      Scrum:
+        description: SCRUM masters
+        maintainers:
+          - durandom
+          - goern
+          - schwesig
+        members:
```
Comment (codificat) [inline github-config.yaml:124]: We need everyone on the team as members of the scrum team...

File: github-config.yaml [hunk 2]
```diff
@@ -115,6 +115,16 @@ orgs:
...
+        privacy: closed
+        repos:
+          scrum: triage
```
Comment (codificat) [inline github-config.yaml:127]: And the team needs more than *triage* permissions to e.g. be able to edit existing issues.

Our usage of the `scrum` repo is not the typical dev repo usage, and editing each other's issues would be expected.

**Review by codificat — CHANGES_REQUESTED** [review]
I just saw this PR after sending #251 (which is already merged), sorry.

I think #251 obsoletes the *Scrum* part of this PR - only leaving the move of Shrey pending.

A rebase will be needed in any case...

**Pull request closed without merging**